Pcap++/header/PcapLiveDevice.h

```diff
@@ -52,8 +52,10 @@ namespace pcpp
 	 */
 	using OnStatsUpdateCallback = std::function<void(IPcapDevice::PcapStats&, void*)>;
 
-	// for internal use only
-	typedef void* (*ThreadStart)(void*);
+	namespace internal
+	{
+		using ThreadStart = void*(*)(void*);
```

Comment:
Could also be represented as: `using ThreadStart = std::add_pointer<void*(void*)>::type;` depending on preference.